src/Network/Probe.php

```diff
@@ -161,7 +168,7 @@ private static function hostMeta($host)
 			return false;
 		}
 
-		if (!is_object($xrd)) {
+		if (!is_object($xrd) && ! empty($url)) {
```

Comment:
Code standards: Please remove the space between the exclamation mark and `empty()`.